.gitignore

```diff
@@ -24,3 +24,4 @@ build.*/
 # Files generated by other shader compilers
 
 *.spv
+/source/slang/core.meta.slang.temp.h
```

Comment:
What's this line for?

Reply:
I don't know why, but slang-generate suddenly start to output this file.

Reply:
Fixed.